guide/14-deep-learning/how_sr3_works.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p> The resulting path <strong>from </strong>from export tool is provided to <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#prepare-data" target="_blank">prepare_data</a> function in <code>arcgis.learn</code> to create a databunch </p><p>Remove from as it appears twice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>